### DISCUSSION TITLE
Contribute a Chinese translation

### DISCUSSION THREAD

**Discussion by Phrixus2023** [lllyasviel/Fooocus#752] [Show and tell]
This is the Chinese translation file, just place it in the language folder to start, no need to modify the bat file, has been synchronized with the official English version of the lllyasviel.Updated to V2.5.0

[default.json](https://github.com/user-attachments/files/16317611/default.json)

**Comment by xhoxye** [comment]
How to use?

**Comment by lllyasviel** [comment]
To use it, rename as `default.json` and put to `Fooocus/language/default.json`

or, put to `Fooocus/language/cn.json` and add `--language cn` to bat

**Reply by lllyasviel** [reply]
note that auto update will not influence user files in `Fooocus/language`

**Reply by Lisofabin** [reply]
how to add  --language cn to bat?Please

**Reply by Lisofabin** [reply]
Thanks, it automatically change to Chinese
